pants.ini

```diff
@@ -388,11 +399,28 @@ verify_commit: False
 
 
 [python-setup]
+interpreter_search_paths: [
+     '<PEXRC>',
+     '<PATH>'
+     # TODO(#7735): We need to add this entry for remoting to be able to discover a valid
+     # interpreter, because <PATH> will refer to the host PATH and not the remote value. This value
+     # was found by inspecting the docker image for remoting.
+     "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/python3.6/bin:/usr/local/go/bin",
```

Comment:
I did not comment this out because it should not cause failures, at least in most instances. If any of these entries do not exist on the machine, then Pex / Pants will just try the other entries in `<PEXRC>` and `<PATH>` (which is the host PATH).

By leaving it in, it's one less thing that users have to change to run remoting.

I see why we might want to comment it out by default, though, and am happy to do so if you think it's worth it.

Reply:
IMO, we should either resolve the platform issue, or (if this is really a good default) submit it as the default to pex.

Reply:
> or (if this is really a good default) submit it as the default to pex.

What do you mean?

> we should either resolve the platform issue

I'd prefer we don't let the platform issue block this PR, because we seem to still be at least a few days away from a good solution. Agreed that it will be great to remove this hack asap.